### ISSUE TITLE
Accessibility Issue - Missing Alt tag on Image and Missing Title Atrributes on iFrames

### ISSUE THREAD

**Issue by lcccweb** [braintree/braintree-web#732]
Various missing attributes impacting accessibility for our website and no doubt many other websites on the internet.

image for a counter is missing an alt attribute, and several iFrames for the checkout process are missing title attributes.

- SDK version: 3.32.0
- Environment: Both
- Browser and OS: Chrome: Version 129.0.6668.101 (Official Build) (64-bit), Windows 11 Pro

Due to upcoming enforcement of WCAG 2.2 Guidelines we have been working to make our website fully accessible.  We have several Gravity Forms on our site that use the PayPal Checkout extension (https://www.gravityforms.com/add-ons/paypal-checkout/) there are three accessibility issues upon initial page load for the form.

Missing Alt Tag for Image -> <img id="baas-iframe-async" src="https://b.stats.paypal.com/counter.cgi?i=127.0.0.1&amp;p=68707999d00f6e7418078ac8c72c6ee9&amp;t=1731012613.797&amp;a=14">

Missing Title Attributes for iFrames

<iframe name="ppfniframe" id="ppfniframe" 

<iframe src="https://assets.braintreegateway.com/web/3.32.0-payments-sdk-dev/html/hosted-fields-payments-sdk-tokenization-frame.min.html" frameborder="0" allowtransparency="true" scrolling="no" name="hosted-fields-tokenization-frame_

If a user select the Credit Card checkout through the iFrame on our site, then each of the fields 
Card Number, Expiration Date, CVV,  and CardHolder Name each are in an iFrame that is also missing a title attribute.

We have not found a way to dynamically add one in via JavaScript or jQuery.  If these missing attribute issues could be solved then this would be less accessibility problems for us many other websites across the internet.
